starter-code/index.js

```diff
@@ -1,11 +1,46 @@
 class SortedList {
-  constructor() {}
-  add(item) {}
-  get(pos) {}
-  max() {}
-  min() {}
-  average() {}
-  sum() {}
+  constructor() {
+    this.items = [];
+    this.length = this.items.length;
+  }
+  add(item) {
+    this.items.push(item);
+    this.length = this.items.length;
+    this.items.sort(function(a,b){return a-b})
+  }
+  get(pos) {
+    return this.items[pos-1];
+  }
+  max() {
+    if(this.length>0){
+      return this.items[this.length-1]
+    }
+    else {throw new Error("EmptySortedList")
+  }
+  }
+  min() {
+    if(this.length>0){
+      return this.items[0]
+    }
+    else {throw new Error("EmptySortedList")
+  }
+  }
+  sum() {
```

Comment:
You could have used a reduce here. It makes it clearer to people reading your code that you're adding up the values to get a single value. 

```
 return this.items.reduce(function(acc, el) {
      return acc + el;
    }, 0);
```